scripts/copyFiles.mjs

```diff
@@ -31,6 +34,10 @@ async function addLicense(packageData) {
   );
 }
 
+async function removeBuildArtefacts() {
```

Comment:
```suggestion
async function removeBuildArtifacts() {
```

We tend to use American spelling throughout the codebase (https://grammarist.com/spelling/artefact-artifact/)